website/styles/spacing.tsx

```diff
@@ -0,0 +1,168 @@
+import { css } from 'emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const margin = {
```

Comment:
can this be generated more programatically?

Reply:
we could have a script that reads in the values from `spacing` and generates this entire file. this is more along the lines of what tailwind does